classes/identity/Identity.inc.php

```diff
@@ -44,51 +44,57 @@ function getFullName($lastFirst = false) {
 	/**
 	 * Get first name.
 	 * @return string
+	 * @param $locale string
 	 */
-	function getFirstName() {
-		return $this->getData('firstName');
+	function getFirstName($locale = null) {
+		return $this->getData('firstName',$locale);
 	}
 
 	/**
 	 * Set first name.
 	 * @param $firstName string
+	 * @param $locale string
 	 */
-	function setFirstName($firstName) {
-		$this->setData('firstName', $firstName);
-	}
+        function setFirstName($firstName, $locale = null) {
```

Comment:
(There are some space-based indents here -- please set your editor to use tabs.)